Components/PalindomeChecker.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+
+export const PalindomeChecker = () => {
+  const [input, setInput] = useState('');
+  const [isPalindrome, setIsPalindrome] = useState(false);
+
+  function isArrayLengthEven (arrLength: number) {
+    if (arrLength % 2 === 0){
+      return true;
+    }
+      return false;
+  }
+
+  const isItAPalindrome = (): void => {
+      let arrToCheck = input.split('');
+      let numOfChecks;
```

Comment:
Maybe we could skip a step by initializing the default variable to something and only reassign if necessary? Or better yet - does the numOfChecks matter?